contracts/ERC721M.sol

```diff
@@ -535,18 +505,6 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         emit PermanentBaseURI(_currentBaseURI);
     }
 
-    /**
-     * @dev Returns token URI suffix.
-     */
-    function getTokenURISuffix()
```

Comment:
we need this, right?

Reply:
Not quite, we never use this externally.